### PR TITLE
Feat - Adding trash page

### PR DESCRIPTION
**Pull request by tetianaman**
Screenshot 2024-05-17 at 18 36 51

### CODE REVIEW

File: src/components/trash-page/trash-page.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    return (
+        <div className={classNames(styles.root, className)}>
+            <SearchInput
+                className={styles.searchInput}
+                value={searchString}
+                placeholder="Search deleted notes"
+                onChange={setSearchString}
+            />
+            <div className={styles.title}>Trash</div>
```
Comment (ynesteruk) [inline src/components/trash-page/trash-page.tsx]: You can use a section component here, i guess.
And you can wrap it with ScrollArea component

File: src/routes.tsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+import { TrashPage } from './components/trash-page/trash-page';
 
 export const routes: RouteObject[] = [
     {
         path: '/',
         element: <SiteWrapper />,
         children: [
             { index: true, element: <Notes /> },
-            { path: 'deleted', element: <Deleted /> },
+            { path: 'trash', element: <TrashPage /> },
```
Comment (ynesteruk) [inline src/routes.tsx:14]: I like old names more

Comment (tetianaman) [reply src/routes.tsx:14]: By design

File: src/components/trash-page/trash-page.tsx (hunk 1 above)
Comment (tetianaman) [reply src/components/trash-page/trash-page.tsx]: I've added a scroll, but I don't see the thumb. Maybe you can help me to understand why.

Comment (ynesteruk) [reply src/components/trash-page/trash-page.tsx]: We have ScrollArea component in our project. Please use it

File: src/routes.tsx (hunk 2 above)
Comment (ynesteruk) [reply src/routes.tsx:14]: It's only a title

Comment (tetianaman) [reply src/routes.tsx:14]: `TrashPage` is more self-explanatory then `deleted`.
I don't want to change it 😇

File: src/components/trash-page/trash-page.module.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    flex-wrap: wrap;
+    gap: $size16;
+}
+
+.searchInput {
+    margin: 0px auto $size2;
+}
+.scrollable-area {
+    width: 100%;
+    height: 100vh;
```
Comment (ynesteruk) [inline src/components/trash-page/trash-page.module.scss]: Is is a correct style?
Scroll area in your case doesn't take the whole viewport

Comment (tetianaman) [reply src/components/trash-page/trash-page.module.scss]: I've put 600px so far. Ideally I should have calculate height as viewport height - search height - section title height. 

Maybe I can do it better with flex, not sure

Comment (ynesteruk) [reply src/components/trash-page/trash-page.module.scss]: Yes. It can be done with flex or grid

**Pull request merged**